src/app/shared/models/fcmUserDto.ts

```diff
@@ -0,0 +1,11 @@
+export interface FcmUserDto {
+  enrolmentDate: Date
+  fcmToken: string
```

Comment:
Wait, are we transferring fcmToken in a DTO. I don't think this is a good idea. Why do we want to expose the fcmToken to the internet traffic?

Reply:
This is required and is mentioned in the FCM docs. Otherwise, the server has no idea which device/app to send the notification to.